### PR TITLE
Release: add a script to tag the release

### PR DESCRIPTION
**Pull request by pracucci**
#### What this PR does
Adding a script to tag the release.

#### Which issue(s) this PR fixes or relates to

N/A

#### Checklist

- [ ] Tests updated
- [ ] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

**Review by aknuds1 — APPROVED** [review]
LGTM

File: tools/release/tag-release.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# Ask confirmation.
+read -p "You're about to tag the version '${ACTUAL_VERSION}'. Do you want to continue? (y/n) " -n 1 -r
+echo ""
+if [[ ! $REPLY =~ ^[Yy]$ ]]; then
+    echo "Aborting ..."
+    exit 1
+fi
+
+git tag -s "mimir-${ACTUAL_VERSION}" -m "v${ACTUAL_VERSION}"
+git push origin "mimir-${ACTUAL_VERSION}"
```
Comment (pstibrany) [inline tools/release/tag-release.sh:32]: Not everybody has "origin" set to Mimir. For example GitHub CLI tool `gh` uses `upstream` (original repository) and `origin` (your fork) remote names.

Comment (pracucci) [reply tools/release/tag-release.sh:32]: Ok will improve it.

Comment (pracucci) [reply tools/release/tag-release.sh:32]: Addressed here: https://github.com/grafana/mimir/pull/3238

**Pull request merged**